parachains/runtimes/assets/statemint/tests/tests.rs

```diff
@@ -0,0 +1,166 @@
+use frame_support::{
+	assert_noop, assert_ok, traits::PalletInfo, weights::WeightToFee as WeightToFeeT,
+};
+use parachains_common::{AccountId, Balance, StatemintAuraId as AuraId};
+use sp_consensus_aura::AURA_ENGINE_ID;
+pub use statemint_runtime::{
+	constants::fee::WeightToFee, xcm_config::XcmConfig, Assets, Balances, ExistentialDeposit,
+	Runtime, SessionKeys, System,
+};
+use xcm::latest::prelude::*;
+use xcm_executor::traits::WeightTrader;
+
+pub struct ExtBuilder {
+	// endowed accounts with balances
+	balances: Vec<(AccountId, Balance)>,
+	// collators to test block prod
+	collators: Vec<AccountId>,
+}
+
+impl Default for ExtBuilder {
+	fn default() -> ExtBuilder {
+		ExtBuilder { balances: vec![], collators: vec![] }
+	}
+}
+use frame_support::traits::GenesisBuild;
```

Comment:
can't these builders be reused?

Reply:
I can put the builder in common so that all runtimes can access them yes

Reply:
Moved the builders to asset-test-utils package, now they are shared across all statemine runtimes